### PR TITLE
Close Campaigns Added

### PR DESCRIPTION
**Pull request by rori4**
- added humanFriendlyName to market database
- related to AdExNetwork/adex-platform#270

Note:
keep in mind that when testing this the georli-market is running at constantly rewrites the data. This should be tested with localdatabase or another database. One solution that Paunov suggested was to change `.collection('campaigns')` to something else like `.collection('campaigns-test')`

### CODE REVIEW

**Comment by rori4** [comment]
@IvoPaunov This was needed for the close on table upgrade to work

File: status-loop/queryValidators.js [hunk 1]
```diff
@@ -52,6 +52,28 @@ function getStatus (messagesFromAll, campaign, balanceTree) {
 	throw new Error('internal error: no status detected; should never happen')
 }
 
+function getHumanFriendlyName (status, campaign) {
+	if (campaign.status && campaign.status.humanFriendlyName === 'Closed') return 'Closed'
+	switch (status) {
+	case 'Active':
```
Comment (Ivshti) [inline status-loop/queryValidators.js]: shouldn't this be idented? other than that, LGTM

Comment (rori4) [reply status-loop/queryValidators.js]: based on the eslint configuration there shouldn't be any indentation here.
I am always formatting with the eslint config on save. I suggest we copy the eslint config from the platform. What do you think?

Comment (Ivshti) [reply status-loop/queryValidators.js]: ok - leave it as is, then in a separate PR, copy the cfg from the platform and format everything

**Pull request merged**